WordPress/WordPressShareExtension/String+Extensions.swift

```diff
@@ -88,6 +88,15 @@ extension String {
         return self + returnURLString
     }
 
+    /// Returns a Boolean value indicating if this String begins with provided prefix.
+    /// - Parameter prefix: The provided prefix to verify.
+    /// - Parameter options: The provided compare options.
```

Comment:
I checked other documentation around the project, and `provided` was used a few times. I agree with you that it can be improved. I'm going to apply your suggestion. Thank you!